src/test/py/bazel/bzlmod/bazel_overrides_test.py

```diff
@@ -272,6 +275,101 @@ def testGitOverride(self):
     self.assertIn('main function => bbb@1.1', stdout)
     self.assertIn('bbb@1.1 => aaa@1.0 (locally patched)', stdout)
 
+  def testGitOverrideStripPrefix(self):
+    self.writeMainProjectFiles()
+
+    # Update BUILD and main.cc to also call `ddd`.
+    self.ScratchFile(
+        'BUILD',
+        [
+            'cc_binary(',
+            '  name = "main",',
+            '  srcs = ["main.cc"],',
+            '  deps = [',
+            '    "@aaa//:lib_aaa",',
+            '    "@bbb//:lib_bbb",',
+            '    "@ddd//:lib_ddd",',
+            '  ],',
+            ')',
+        ],
+    )
+    self.ScratchFile(
+        'main.cc',
+        [
+            '#include "aaa.h"',
+            '#include "bbb.h"',
+            '#include "ddd.h"',
+            'int main() {',
+            '    hello_aaa("main function");',
+            '    hello_bbb("main function");',
+            '    hello_ddd("main function");',
+            '}',
+        ],
+    )
+    src_aaa_1_0 = self.main_registry.projects.joinpath('aaa', '1.0')
+    src_ddd_1_0 = self.main_registry.projects.joinpath('ddd', '1.0')
+    self.RunProgram(['git', 'init'], cwd=src_aaa_1_0)
+    self.RunProgram(
+        ['git', 'config', 'user.name', 'tester'],
+        cwd=src_aaa_1_0,
+    )
+    self.RunProgram(
+        ['git', 'config', 'user.email', 'tester@foo.com'],
+        cwd=src_aaa_1_0,
+    )
+
+    # Make a subdirectory that itself is the published module 'ddd'.
+    subdir_name = 'subdir_containing_ddd'
+    shutil.copytree(src=src_ddd_1_0, dst=src_aaa_1_0 / subdir_name)
```

Comment:
I could also use `self.RunProgram(["cp", "-r", ...])` if that's better for some reason.

Reply:
No, that may not work on Windows.